app/models/story.rb

```diff
@@ -94,4 +94,14 @@ def release_date=(val)
     date = Chronic.parse(val)
     self[:release_date] = date
   end
+
+  def saved_changes
+    return super unless Array(documents_attributes_was) != documents_attributes
+
+    super.merge(documents_attributes: [documents_attributes_was, documents_attributes])
```

Comment:
If by 'this' you mean the return, it is to prevent the inclusion of 'documents_attributes' on the `saved_changes` response even if there are no changes to the documents, so we check if there are any changes, if there are then we include them.

If you mean the merge, as I explained on the other conversation, saved_changes does not include the changes made to the story's documents, this way we can keep the default behaviour while also including the desired information.